pandas/core/frame.py

```diff
@@ -910,17 +911,35 @@ def itertuples(self, index=True, name="Pandas"):
 
```

Comment:
In the `See Also`, can you make the `A` in the title capital? And then, the items, if you can prefix `iterrows` and `iteritems` with `DataFrame.iterrows`...

Reply:
Ok, just prefixed both `iterrows` and `iteritems`.